modules/src/core/ics04_channel/error.rs

```diff
@@ -324,6 +324,22 @@ define_error! {
                              e.port_channel_id.1)
             },
 
+        ProcessedTimeNotFound
+            { client_id: ClientId }
+            | e | {
+                format_args!(
+                    "Processed time for the client {0} not found",
```

Comment:
Nit: It would be good to capture the specific `Height` in the error type, which would help tracking down bugs or error causes.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1583/commits/7381e02bd64c2c7821380b8de70e180b642d5642